packages/ketcher-macromolecules/src/components/LeftMenuComponent/LeftMenuComponent.tsx

```diff
@@ -33,7 +33,7 @@ export function LeftMenuComponent() {
   const activeMenuItems = [activeTool];
 
   const menuItemChanged = (name) => {
-    editor.events.selectTool.dispatch(name);
+    editor.events.selectTool.dispatch(name, { toolName: name });
```

Comment:
It's a bit confusing why we pass the name of the tool twice, is it due to the nested menu items? Can we avoid it somehow? We still have unique tool names in the end

Reply:
It's because options are passed directly to active tool and then tool can use them. So toolName option is used in Bond tool to create appropriate bond type